com.github.philip_scott.spice-up.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "com.github.philip_scott.spice-up",
-    "base": "io.elementary.Loki.BaseApp",
+    "base": "io.elementary.Juno.BaseApp",
```

Comment:
This will be changed to `io.elementary.BaseApp`. You will need to change the `base-version' to `Juno`